standards/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,19 @@
-_This goes in your project's root directory under `.github/` folder_
+## Pull Request Templates
+
+[Github documentation](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository#adding-a-pull-request-template)
+#### To add an issue template to your repository:
+
+1. Create a file named `PULL_REQUEST_TEMPLATE.md` inside your **.github** folder
+2. Put the markdown that you want for your Pull Request template in this file. Below is an example of one potential template. Feel free to use it for your repository or create your own.
+3. Open a pull request to your default branch. This will only work if the exact file structure and file name referenced in step 1 is in your default branch.
+4. Have your team review and approve your pull request. Once it is closed, you will have pull request templates
```

Comment:
Should we mention that you can tag `PR` team for code reviews? or is that already here and I missed it?

Reply:
Agreed, I think we should mention including the PR Team in both cases. Other than that, this has my seal of approval!

Reply:
Agreed